crates/exec-wasmtime/src/workload.rs

```diff
@@ -42,21 +42,15 @@ pub enum Package {
     Remote(Url),
 
     /// Local package
-    #[cfg(unix)]
     Local {
         /// Open WASM module file descriptor
+        #[cfg(unix)]
         wasm: std::os::unix::prelude::RawFd,
-        /// Optional open config file descriptor
-        conf: Option<std::os::unix::prelude::RawFd>,
-    },
-
-    /// Local package
-    #[cfg(windows)]
-    Local {
         /// Open WASM module file
+        #[cfg(windows)]
         wasm: std::fs::File,
-        /// Optional open config file
-        conf: Option<std::fs::File>,
+        /// Enarx configuration
+        config: Config,
```

Comment:
Why rename this field?
Also note that @npmccallum likes variables/fields to be the same character length